### ISSUE TITLE
principal_nodal_stress(rnum) etc API suggestion

### ISSUE THREAD

**Issue by 1990chs** [pyansys/pymapdl-reader#52]
principal_nodal_stress(rnum) this API can't assign the nodes numbers.  It needs update.
I have check the API document and I found some API can't assign nodes numbers or component.
The list is follow.
```
principal_nodal_stress(rnum)
element_stress(rnum, principal=False, in_element_coord_sys=False, **kwargs)
nodal_acceleration(rnum, in_nodal_coord_sys=False)
nodal_boundary_conditions(rnum)
nodal_input_force(rnum)
nodal_reaction_forces(rnum)
nodal_time_history(solution_type='NSL', in_nodal_coord_sys=False)
nodal_velocity(rnum, in_nodal_coord_sys=False)

class ansys.mapdl.reader.cyclic_reader.CyclicResult(filename)
nodal_displacement(rnum, phase=0, full_rotor=False, as_complex=False)
nodal_elastic_strain(rnum, phase=0, as_complex=False, full_rotor=False)
nodal_plastic_strain(rnum, phase=0, as_complex=False, full_rotor=False)
nodal_solution(rnum, phase=0, full_rotor=False, as_complex=False)
nodal_stress(rnum, phase=0, as_complex=False, full_rotor=False)
nodal_thermal_strain(rnum, phase=0, as_complex=False, full_rotor=False)
nodal_temperature(rnum, full_rotor=False)```

**Comment by akaszynski** [comment]
Thanks for posting this @1990chs.

We just had a baby, so it might take me longer than I'd like to patch this bug.  I should have something by Monday next week.

**Comment by 1990chs** [comment]
Congratulation. It is OK

**Comment by 1990chs** [comment]
I try to edit the rst.py as follow, do I modify is correct?
![image](https://user-images.githubusercontent.com/27892759/124103246-a9dfdb80-da93-11eb-9342-2c806ad42583.png)
![image](https://user-images.githubusercontent.com/27892759/124103267-b106e980-da93-11eb-9ba3-62908ecda23a.png)

**Comment by akaszynski** [comment]
If you fork and then create a PR, I'll be able to review your work and make suggestions.

**Comment by 1990chs** [comment]
What's the PR? I am not a professional programmer.

**Comment by akaszynski** [comment]
It's ok, I'm not either :)

PR is a pull request.  There's tons of examples on forking a repo and making a pull request.  Once you figure it out the first time it's not too bad.

**Comment by 1990chs** [comment]
Thinks,yesterday I try to use vs code to push codes to my github .but not successed . Today  I use  git bash to push and successed. I have ask a pull request

**Comment by jgd10** [comment]
This appears to have been fixed by #56 ? If so is there anything else needed or can we close the issue?

**Comment by jgd10** [comment]
Since this appears to be fixed I'll close this for now and we can reopen if something was missed!
